### PR TITLE
WRP-3602: Fix VirtualList to keep focus when dataSize is reduced

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
The focus could be lost if a focused item in `VirtualList` is removed by reduced `dataSize`.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fix to focus the last item of a list in a case of that the last focused item is removed by decreasing `dataSize`.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3602

### CODE REVIEW

File: samples/qa-virtualgridlist-datasize-changing/webos-meta/appinfo.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "id": "enact.perf.virtualgridlist.native",
```
Comment (juwonjeong) [inline samples/qa-virtualgridlist-datasize-changing/webos-meta/appinfo.json:2]: I think it would be better to update id and title to qa-~.

Comment (0x64) [reply samples/qa-virtualgridlist-datasize-changing/webos-meta/appinfo.json]: 😨 😓 👌 🙇

**Review by juwonjeong — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**